utils/v2/connector/impl/CustomRequestOnDemand.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  SignedPriceDataType,
+} from "redstone-api-extended/lib/oracle/redstone-data-feed";
+import { PriceFeedConnector } from "../PriceFeedConnector";
+import axios from "axios";
+import EvmPriceSigner from "redstone-node/dist/src/signers/EvmPriceSigner";
+
+export interface CustomRequestDetails {
+  url: string;
+  jsonpath: string;
+  expectedSymbol: string;
+};
+
+export interface CustomRequestOnDemandOptions {
+  nodes: string[];
```

Comment:
Why this is array when we always use the first node?

Reply:
Good point. This is for future support, when we will request data from multiple nodes